### PR TITLE
docs(readme): fixed typo

### PR DESCRIPTION
**Pull request by JLou**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,4 +50,4 @@ with a workflow :
 
 - Going Open Source brings and requires a change of mentality within development teams.
 
-![Worflow step 4](./images/workflow-step4.png)
\ No newline at end of file
+![Worflow step 4](./images/workflow-step4.png)
```
Comment (guillaume-chervet) [inline README.md]: Worklfow?

Thank you gor your pr @JLou  😀

Comment (JLou) [reply README.md]: I didn't touch that line, it's the absence of new line at the end making git confused. I can fix it though

Comment (JLou) [reply README.md]: I fixed all the instances of _worflow_. Should be good to merge now!

**Review by guillaumechervetaxa — APPROVED** [review]
Thank you

**Comment by guillaumechervetaxa** [comment]
@JLou souhaites-tu être repository maintainer du repo?

**Pull request merged**